client/src/containers/Explore.js

```diff
@@ -217,11 +239,73 @@ export const Explore = () => {
             .catch(error => console.log(error));    
     };
 
+    if (width < 840){ // mobile view
     return(
         <Wrapper>
         <Navbar />
         <main>
             <PageWrapper>
+                <AdBoxMobile>
+                    <adCarrier
+                    Height={50}
+                    Width={328}
+                    Path="cds_leaderboard"
+                    />
+                </AdBoxMobile>
+                <TextWrapper>
+                    <h1>Explore Clubs</h1>
+                    <p>Find your Columbia community</p>
+                </TextWrapper>
+                <FiltersBox>
+                    <FiltersLeft>
+                        <SearchBox><SearchBar></SearchBar></SearchBox>
+                        <Filter><Type /></Filter>
+                        <Filter><Size /></Filter>
+                        <Filter><Join /></Filter>
+                    </FiltersLeft>
+                    <ShuffleBox>
+                        <ShuffleButton onClick={newFetch}>
+                            <ShuffleImage><img src={Icon} width={15} height={15} alt="shuffle" /></ShuffleImage>
+                            <ShuffleWord>Shuffle</ShuffleWord>
+                        </ShuffleButton>
+                    </ShuffleBox>
+                </FiltersBox>
+                <CardsContainer>
+                    {(clubProfiles.length === 0) ? (<h1>Loading</h1>) : (clubProfiles.map(profile => (
+                        <CardWrapper>
+                            <ExploreBox 
+                                name = {profile.name}
+                                description = {profile.shortDescription}
+                                imageURL = {profile.imageUrl}
+                                tags = {profile.tags}
+                                clubSize = {profile.memberRange}
+                                acceptingMembers = {profile.acceptingMembers}
+                                applicationRequired = {profile.applicationRequired}
+                                cardLink={`/club/${profile._id}`}
+                            />
+                        </CardWrapper>
+                    )))} 
+                </CardsContainer>  
+                <BottomWrapper>
+                    <FilterBottom><FilterMobile /></FilterBottom>
+                </BottomWrapper>
+            </PageWrapper>
+        </main>
+        </Wrapper>
+    )
+    } else { //desktop view
```

Comment:
You can delete all of this extra code below since the AdBoxType handles both the mobile and desktop AdBox you styled!!